### ISSUE TITLE
Sample request parameter types are shown with paragraph tags

### ISSUE THREAD

**Issue by crackalak** [apidoc/apidoc#300]
![apidoc1](https://cloud.githubusercontent.com/assets/4701936/8570064/a1e1cb70-2575-11e5-9d97-2594789356a8.png)

The header types are stored in the api_data.json without html tags like so:
```
{
    "group": "Header",
    "type": "String",
    "optional": false,
    "field": "Authorization",
    "description": "<p>Authorization value</p> "
}
```

and parameter types stored with html tags:
```
{
     "group": "Parameter",
     "type": "<p>Date</p> ",
     "optional": true,
     "field": "modified",
     "description": "<p>Last Modified Date</p> "
}
```

This also causes the type in the header list to be displayed in black font colour rather than the parameter type in the grey:

![apidoc2](https://cloud.githubusercontent.com/assets/4701936/8570229/eaafe53e-2576-11e5-802d-08e596fb8b51.png)

**Comment by wisoot** [comment]
I haven't looked much into your code yet, but I think it is due to markdown parser is used on parameter type, which it shouldn't be.

**Comment by travisdavis75** [comment]
@wisoot nailed it.  This change to apidoc-core/lib/parsers/api_[errors|param|success].js removes the paragraph tags:
-    markdownFields: [ 'description', 'type' ]
+    markdownFields: [ 'description' ]

**Comment by ianclegg** [comment]
great work @travisdavis75 and @wisoot - can we expect a PR?

**Comment by travisdavis75** [comment]
PR submitted.

**Comment by wisoot** [comment]
Thanks @travisdavis75, But, you have to fix the test as well.

I'm curious is this project still being maintained?

**Comment by ianclegg** [comment]
I'm not sure the problem is with the test, the paragraph tags were added to the test in a recent change:
https://github.com/apidoc/apidoc-example/commit/96dd825fa69fa87a913b469ad619a05f3a79dccb
so that would imply the code and test being wrong - not saying it doesn't happen, but it is unusual. Perhaps that tags are meant to be there, but the issue is that they are html escaped?
@rottmann can you spare a minute to clarify this?

**Comment by ianclegg** [comment]
Looks like the tags are not there on the example: http://apidocjs.com/example/
and un-escaping them breaks the formatting, so I guess they shouldn't be there - which would imply a bad test?

**Comment by wisoot** [comment]
This is what spit out from travis:
data not equals to node_modules/apidoc-example/v0.2.0/fixtures/api_data.json in line 247
fixture:             "type": "<p>String</p> ",
created:             "type": "String",

**Comment by wisoot** [comment]
This is nothing to do with this project per se, but with apidoc-example

**Comment by wisoot** [comment]
I think I'm too quick to point out the problem. Upon looking closer into apidoc-example, I see this:

https://github.com/apidoc/apidoc-example/blob/master/v0.2.0/src/api_param.js

Line 43: @apiParam {[Custom]\(http://en.wikipedia.org/wiki/Data_type)} param29 Type with markdown link.

So type should support markdown as well according to the intention. But for some reason when spit out into html those tags are encoded into \&lt;p\&gt;

**Comment by ianclegg** [comment]
@wisoot technically none of this is to do with this specific project/repo - the patch @travisdavis75 made was in apidoc-core - and the test fixtures for apidoc-core are in apidoc-example. Take a closer look.
The point I was making is that we don't normally change a test unless we are sure its wrong, more often than not the code is wrong and not the test. So @rottmann put the paragraph tags in the test fixture for a reason

**Comment by ianclegg** [comment]
I agree, so we get html encoded tags, but even if we fix that and get correctly rendered paragraph tags - it'll break the formatting - that's where I get confused with the that commit

**Comment by wisoot** [comment]
I think the mistake is in handlebar tag rather then {{type}} should be --> {{{type}}}
So it doesn't escape html

**Comment by travisdavis75** [comment]
I @wisoot is correct and as pointed out there is already a pull request for this change.

**Comment by ianclegg** [comment]
Well that sorts the default template, but those paragraphs broke my bootstrap 3 css

I still cant figure out why those paragraph tags they are even there in the first place. There is absolutely no markdown in the @apiparam paramtype - so the type string should pass through unaltered. My hunch is that the underlying problem is that the markdown parser is not setup correctly

**Comment by ianclegg** [comment]
Anyway, you've definitely fixed a bug there @wisoot - and needs to to be merged

Since its technically unrelated, I'll open an new issue questioning why the paragraph tags are there in the first place

**Comment by cnlinjie** [comment]
me too~
I have the problem too。
- -!

**Comment by DannyvdSluijs** [comment]
Im guessing #270 and #358 are the same issue. I triggered on issue 270. But there seems to be no solution merged.

**Comment by rottmann** [comment]
Fix in 0.13.2

**Comment by hareesh-blippar** [comment]
This is still broken as of 0.14.0
